packages/block-library/src/list/v2/edit.js

```diff
@@ -126,16 +127,22 @@ function IndentUI( { clientId } ) {
 	);
 }
 
-function Edit( { attributes, setAttributes, clientId } ) {
-	const blockProps = useBlockProps();
+function Edit( { attributes, setAttributes, clientId, style } ) {
+	const blockProps = useBlockProps( {
+		...( Platform.isNative && { style } ),
+	} );
 	const innerBlocksProps = useInnerBlocksProps( blockProps, {
 		allowedBlocks: [ 'core/list-item' ],
 		template: TEMPLATE,
 		templateInsertUpdatesSelection: true,
+		...( Platform.isNative && { marginVertical: 8, marginHorizontal: 8 } ),
```

Comment:
What is the default here? Why is this set with inner block props and not CSS?

Reply:
The default values are `16` for both margins, this is used in the [BlockList](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/block-list/index.native.js#L246-L247) to calculate some block spacings. We do something similar for the [Gallery block](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/gallery/gallery.native.js#L68-L69), maybe I could move the `8` value to a variable so we don't have a random number there. What do you think?